Octokit.Reactive/Clients/IObservableTeamsClient.cs

```diff
@@ -12,6 +12,11 @@ namespace Octokit.Reactive
     /// </remarks>
     public interface IObservableTeamsClient
     {
+        /// <summary>
+        /// Returns a client to manage discussions for a team.
+        /// </summary>
+        IObservableTeamDiscussionsClient TeamDiscussion { get; }
```

Comment:
We want to access this API like `client.Team.Discussion.Get()` so this property name should just be `Discussion`

```suggestion
        IObservableTeamDiscussionsClient Discussion { get; }
```